extensions/podman/package.json

```diff
@@ -170,6 +170,12 @@
           "default": false,
           "description": "Docker compatibility mode is a feature that allows you to use Podman as a drop-in replacement for Docker-compatible CLI tools. Podman will automatically emulate a Docker API socket connection (typically) to /var/run/docker.sock. NOTE: This requires administrative privileges.",
           "when": "isMac"
+        },
+        "podman.setting.rosetta": {
+          "type": "boolean",
+          "default": true,
+          "description": "Use macOS Rosetta to run x86_64 Linux binaries, disabling will use qemu. NOTE: Recreating the Podman Machine VM is required for the changes to take effect.",
```

Comment:
'NOTE: Recreating the Podman Machine VM is required for the changes to take effect.' is a bit odd if we don't know if you have a machine, or whether you used the setting. Maybe 'NOTE: This setting is only applied when creating machines.' or 'NOTE: This setting is only applied to new machines.'?